let graphData={graphOptions:{attractionForce:1,linkLength:10,repulsionForce:150,centralForce:3,edgePruning:100,minNodeRadius:3,maxNodeRadius:7},isInitialized:!0,paths:["notes/bioinformatics/introdução-à-bioinformática/supplemental/colinearity-and-transcription-units.html","notes/bioinformatics/introdução-à-bioinformática/algorithms-for-bioinformatics.html","notes/bioinformatics/introdução-à-bioinformática/annotated-genome-assemblies,-how-to-get-there_.html","notes/bioinformatics/introdução-à-bioinformática/bibliographic-resources.html","notes/bioinformatics/introdução-à-bioinformática/biological-sequences.html","notes/bioinformatics/introdução-à-bioinformática/deep-ai-what-are-the-main-differences-between-aligning-multiple-sequences-with-tcoffee,-muscle-or-dialign_.html","notes/bioinformatics/introdução-à-bioinformática/deep-ai-knowing-about-prompt-engineering.html","notes/bioinformatics/introdução-à-bioinformática/deep-ai-sequence-evolution,-associated-aspects.html","notes/bioinformatics/introdução-à-bioinformática/deepai-how-are-the-substitution-scores-calculated_.html","notes/bioinformatics/introdução-à-bioinformática/deepai-how-does-dynamic-programming-works-in-the-nedleman-wunsch-algorithm_.html","notes/bioinformatics/introdução-à-bioinformática/deepai-how-should-one-read-the-matrix-values_.html","notes/bioinformatics/introdução-à-bioinformática/deepai-what-about-the-needleman-wunsch-without-end-gap-penalties_.html","notes/bioinformatics/introdução-à-bioinformática/deepai-what-does-62-means-in-blosum62_.html","notes/bioinformatics/introdução-à-bioinformática/deepai-what-does-120-means-in-pam120_.html","notes/bioinformatics/introdução-à-bioinformática/deepai-what-is-a-default-substitution-matrix-and-why-do-we-need-it_.html","notes/bioinformatics/introdução-à-bioinformática/deepai-what-is-a-substitution-matrix_.html","notes/bioinformatics/introdução-à-bioinformática/deepai-what-is-the-guide-tree-in-the-context-of-the-clustal-algorithm_.html","notes/bioinformatics/introdução-à-bioinformática/deepai-what-is-the-main-difference-between-the-smith-waterman-versus-the-needleman-wunsch-algorithms_.html","notes/bioinformatics/introdução-à-bioinformática/deepai-why-dynamic-programming-cannot-be-used-for-multiple-sequence-alignment_.html","notes/bioinformatics/introdução-à-bioinformática/dot-plot.html","notes/bioinformatics/introdução-à-bioinformática/functional-annotation,-the-gate-to-structure-and-function.html","notes/bioinformatics/introdução-à-bioinformática/hands-on-class-ap02-algorithms-in-action.html","notes/bioinformatics/introdução-à-bioinformática/hands-on-class-ap03-algorithms-in-action.html","notes/bioinformatics/introdução-à-bioinformática/hands-on-session_pairwise-alignment.html","notes/bioinformatics/introdução-à-bioinformática/hands-on-session_proteins,-sequences-&-algorithms.html","notes/bioinformatics/introdução-à-bioinformática/hands-on-session,-public-database-servers.html","notes/bioinformatics/introdução-à-bioinformática/hands-on-session,-evolutionary-distance-&-evolution-models.html","notes/bioinformatics/introdução-à-bioinformática/hands-on-session,-multiple-sequence-analysis.html","notes/bioinformatics/introdução-à-bioinformática/hands-on-session,-reaching-for-structure-and-function.html","notes/bioinformatics/introdução-à-bioinformática/hands-on-session,-which-cross-reference-is-that_.html","notes/bioinformatics/introdução-à-bioinformática/hmm-hidden-markov-models.html","notes/bioinformatics/introdução-à-bioinformática/introdução-à-bioinformática.html","notes/bioinformatics/introdução-à-bioinformática/multiple-sequence-alignment,-considerations-on-the-results.html","notes/bioinformatics/introdução-à-bioinformática/multiple-sequence-alignments,-coding-for-purpose.html","notes/bioinformatics/introdução-à-bioinformática/ontologies-&-knowledge-representation.html","notes/bioinformatics/introdução-à-bioinformática/pairwise-alignment,-wrapping-up.html","notes/bioinformatics/introdução-à-bioinformática/pairwise-alignment,-the-dynamic-programming-role.html","notes/bioinformatics/introdução-à-bioinformática/program.html","notes/bioinformatics/introdução-à-bioinformática/protein-sequence-analysis.html","notes/bioinformatics/introdução-à-bioinformática/protein-sequence-databases,-a-game-of-twins.html","notes/bioinformatics/introdução-à-bioinformática/protocolo_1_registo_armazenamento_sequências.html","notes/bioinformatics/introdução-à-bioinformática/relational-databases.html","notes/bioinformatics/introdução-à-bioinformática/sequence-alignment.html","notes/bioinformatics/introdução-à-bioinformática/sequence-evolution.html","notes/bioinformatics/introdução-à-bioinformática/sequence-evolution,-associated-aspects.html","notes/bioinformatics/introdução-à-bioinformática/sequence-pairwise-alignment-fast-algorithms,-heuristics-to-the-rescue.html","notes/bioinformatics/introdução-à-bioinformática/studying-by-the-bot.html","notes/bioinformatics/introdução-à-bioinformática/the-cornerstones-of-bioinformatics,-where-the-data-dwells.html","notes/bioinformatics/introdução-à-bioinformática/the-global-nucleotide-sequence-database.html","notes/bioinformatics/introdução-à-bioinformática/the-substitution-matrix.html","notes/bioinformatics/introdução-à-bioinformática/translated-regions-detection.html","notes/bioinformatics/introdução-à-bioinformática/when-aligning-more-than-two-sequences.html"],nodeCount:52,linkSources:[0,1,1,1,2,2,3,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,20,21,21,22,22,23,23,24,24,25,25,26,26,27,27,28,28,29,30,30,32,32,33,33,34,35,35,36,36,37,37,37,37,37,37,37,37,37,37,37,37,37,37,37,37,37,37,37,37,37,37,37,37,37,37,37,37,37,37,37,37,38,38,39,39,40,41,42,42,43,44,44,45,45,46,46,46,46,46,46,46,46,46,46,46,46,46,46,47,48,48,49,49,49,50,50,51,51],linkTargets:[50,37,4,19,37,47,37,47,37,46,46,46,10,46,15,17,14,14,15,46,46,9,46,1,37,33,37,1,1,37,37,35,37,38,37,47,37,44,37,32,37,20,37,37,38,37,51,37,32,37,37,45,37,42,31,34,4,41,40,1,50,21,38,43,42,24,49,36,45,35,23,51,32,27,44,26,20,33,47,3,48,39,25,2,29,46,37,30,37,48,37,37,37,43,37,37,43,37,36,37,6,15,10,8,14,12,13,9,11,18,16,5,7,37,37,3,37,42,43,1,37,37,35],labels:["Colinearity and Transcription Units","Algorithms for Bioinformatics","Annotated Genome Assemblies, how to get there_","Bibliographic Resources","Biological Sequences","Deep  AI - What are the main differences between aligning multiple sequences with Tcoffee, Muscle or Dialign_","Deep AI - Knowing about Prompt Engineering","Deep AI - Sequence Evolution, Associated Aspects","DeepAI - How are the substitution scores calculated_","DeepAI - How does dynamic programming works in the Nedleman-Wunsch algorithm_","DeepAI - How should one read the matrix values_","DeepAI - What about the Needleman-Wunsch without end-gap penalties_","DeepAI - What does 62 means in BLOSUM62_","DeepAI - What does 120 means in PAM120_","DeepAI - What is a default substitution matrix and why do we need it_","DeepAI - What is a substitution matrix_","DeepAI - What is the guide tree in the context of the CLUSTAL algorithm_","DeepAI - What is the main difference between the Smith-Waterman versus the Needleman-Wunsch algorithms_","DeepAI - Why dynamic programming cannot be used for multiple sequence alignment_","Dot-Plot","Functional Annotation, the gate to Structure and Function","Hands-on Class - AP02 - Algorithms in Action","Hands-on Class - AP03 - Algorithms in Action","Hands-on Session_Pairwise Alignment","Hands-on Session_Proteins, sequences & algorithms","Hands-on Session,  Public Database Servers","Hands-On Session, Evolutionary Distance & Evolution Models","Hands-On Session, Multiple Sequence Analysis","Hands-On Session, Reaching for Structure and Function","Hands-on Session, Which cross-reference is that_","HMM - Hidden Markov Models","Introdução à Bioinformática","Multiple Sequence Alignment,  considerations on the results","Multiple Sequence Alignments, coding for purpose","Ontologies & Knowledge Representation","Pairwise Alignment, wrapping up","Pairwise-Alignment, the Dynamic Programming Role","Program","Protein Sequence Analysis","Protein Sequence Databases, a game of twins","Protocolo_1_Registo_Armazenamento_Sequências","Relational Databases","Sequence Alignment","Sequence Evolution","Sequence Evolution, Associated Aspects","Sequence Pairwise-Alignment Fast Algorithms, Heuristics to the Rescue","Studying by the Bot","The Cornerstones of Bioinformatics, where the data dwells","The Global Nucleotide Sequence Database","The Substitution Matrix","Translated Regions Detection","When Aligning More Than Two Sequences"],radii:[3.1500803489439857,4.1202938475665745,3.4416322314049586,3.5831037649219466,3.4416322314049586,3.2972910927456383,3.2972910927456383,3.2972910927456383,3.2972910927456383,3.4416322314049586,3.4416322314049586,3.2972910927456383,3.2972910927456383,3.2972910927456383,3.5831037649219466,3.5831037649219466,3.2972910927456383,3.2972910927456383,3.2972910927456383,3.2972910927456383,3.5831037649219466,3.4416322314049586,3.2972910927456383,3.4416322314049586,3.4416322314049586,3.4416322314049586,3.4416322314049586,3.4416322314049586,3.2972910927456383,3.2972910927456383,3.4416322314049586,3.1500803489439857,3.7217056932966024,3.5831037649219466,3.2972910927456383,3.7217056932966024,3.5831037649219466,7,3.7217056932966024,3.4416322314049586,3.2972910927456383,3.2972910927456383,3.7217056932966024,3.7217056932966024,3.5831037649219466,3.5831037649219466,5.638888888888889,3.7217056932966024,3.5831037649219466,3.5831037649219466,3.5831037649219466,3.5831037649219466],linkCount:123}